modules/infrastructure-moving-logging.adoc

```diff
@@ -184,12 +178,12 @@ cluster-logging-operator-84d98649c4-zb9g7       1/1     Running       0
 elasticsearch-cdm-hwv01pf7-1-56588f554f-kpmlg   2/2     Running       0          28m
 elasticsearch-cdm-hwv01pf7-2-84c877d75d-75wqj   2/2     Running       0          28m
 elasticsearch-cdm-hwv01pf7-3-f5d95b87b-4nx78    2/2     Running       0          28m
-fluentd-42dzz                                   1/1     Running       0          28m
-fluentd-d74rq                                   1/1     Running       0          28m
-fluentd-m5vr9                                   1/1     Running       0          28m
-fluentd-nkxl7                                   1/1     Running       0          28m
-fluentd-pdvqb                                   1/1     Running       0          28m
-fluentd-tflh6                                   1/1     Running       0          28m
+collector-42dzz                                   1/1     Running       0          28m
```

Comment:
```suggestion
collector-42dzz                                 1/1     Running       0          28m
```
Looks like these have a few extra spaces and aren't aligning starting with the pod count.